### PR TITLE
Vol1 vol2 iti xds cp iti 1301

### PR DESCRIPTION
**Pull request by Ben-Levy-62**
## 📑 CP Integration CP-ITI-1301

### CODE REVIEW

**Comment by Ben-Levy-62** [comment]
In ITI-18, the li at 2454 does not have a /li to terminate it.  When I add one at 2468, all the surrounding list stuff breaks.  I also changed "XDS Registrys" to "XDS Registries".

**Comment by JohnMoehrke** [comment]
> In ITI-18, the li at 2454 does not have a /li to terminate it. When I add one at 2468, all the surrounding list stuff breaks. I also changed "XDS Registrys" to "XDS Registries".

I think I fixed this.

**Comment by JohnMoehrke** [comment]
@Ben-Levy-62 I further refined the bullets in that section. The encoding was really difficult to follow, it is now simple nesting. I started at 1 because number at zero is roman numerals.

File: ITI/TF/Volume1/ch-10.html [hunk 1]
```diff
@@ -747,6 +753,13 @@ <h3 id="10.2.10">10.2.10 Delayed Document Assembly Option</h3>
       </ul>
       <p>See <a href="../Volume2/ITI-42.html#3.42.4.1.2.1">ITI TF-2:3.42.4.1.2.1</a> and <a href="../Volume2/ITI-43.html#3.43.4.1.3.2">ITI TF-2:3.43.4.1.3.2</a></p>
       <p>In order for an Integrated Document Source/Repository to support the Delayed Document Assembly Option it must interact with a Document Registry supporting the Document Metadata Update Option.</p>
+      <h3 id="10.2.11">10.2.11 Target Communities Option</h3>
+      <p>The Document Consumer which supports the Target Communities Option:</p>
+      <ul>
+         <li class="bullet-list0">In Registry Stored Queries which require a patient identifier, can specify the “$targetCommunityIdList” parameter, containing the list of the communities to which it wants to direct the request to.</li>
+	 <li class="bullet-list0">In queries in which the “$targetCommunityIdList” is specified, the Document Consumer shall not use the “$home” parameter.</li>
```
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-10.html:760]: in the CP this bullet says:
In queries in which the “$targetCommunityIdList” is specified, the Document Consumer shall not populate the “home” attribute on the AdhocQuery element.

File: ITI/TF/Volume1/ch-10.html [hunk 2]
```diff
@@ -747,6 +753,13 @@ <h3 id="10.2.10">10.2.10 Delayed Document Assembly Option</h3>
...
+      </ul>
+      <p>Note that when addressing the query to the Initiating Gateway, the Document Consumer that specifies the “$targetCommunityIdList” parameter must be aware that the Initiating Gateway may not support the “Target Communities” Option. See Section 3.18.4.1.2.5 Compatibility of Options.</p>
```
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-10.html:762]: i suggest to put a reference on see section 3.18.4.1.2.5

e.g. See Section <a href="../Volume2/ITI-18.html#3.18.4.1.2.5">ITI TF-2:3.18.4.1.2.5</a> Compatibility of Options.

File: ITI/TF/Volume1/ch-10.html (hunk 1 above)
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-10.html:760]: in the CP the bullet says:
In queries in which the “$targetCommunityIdList” is specified, the Document Consumer shall not populate the “home” attribute on the AdhocQuery element.

File: ITI/TF/Volume1/ch-10.html (hunk 2 above)
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-10.html:762]: i suggest to put a reference on see section 3.18.4.1.2.5

e.g. See Section [ITI TF-2:3.18.4.1.2.5](https://github.com/IHE/publications/pull/Volume2/ITI-18.html#3.18.4.1.2.5) Compatibility of Options.

File: ITI/TF/Volume2/ITI-18.html [hunk 3]
```diff
@@ -879,6 +895,10 @@ <h6 id="3.18.4.1.2.3">3.18.4.1.2.3 Query Request Parameters – Coding Style</h6
                     <sup>2</sup>
                     Shall be coded according to specification in <a href="#3.18.4.1.2.3.4">Section 3.18.4.1.2.3.4 Coding of Code/Code-Scheme</a>.
                 </p>
+                <p>
+                    <sup>3</sup>
+                    The use of this optional parameter is only expected if the Document Consumer supports the “Target Communities” Option (See Section 10.2.11 Target Communities Option).
+                </p>
                 <p class="heading8" id="3.18.4.1.2.3.7.3">3.18.4.1.2.3.7.3 FindFolders</p>
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-18.html:902]: FindFolders adHocQuery is missing the $targetCommunityIdList parameter in the table. line 948

File: ITI/TF/Volume2/ITI-18.html [hunk 4]
```diff
@@ -2033,26 +2088,56 @@ <h6 id="3.18.4.1.2.4">3.18.4.1.2.4 Stored Query IDs</h6>
...
+                        on the Document Registry. If this parameter is specified, and the Document Registry does not support
+                        it, the Document Registry shall ignore. If it is specified, and the Document Registry does support
+                        it, the proper information is returned.</p>
+                        <ul>
+                            <li class="bullet-list1">If the Document Consumer does not support the On-Demand Documents Option,
+                                it will send a Registry Stored Query request which does not contain the $XDSDocumentEntryType
+                                parameter. The Document Registry will therefore not supply any On-Demand Document Entries in the
+                                query response.
+                            </li>
+                            <li class="bullet-list1">If the Document Consume does support the On-Demand Documents Option<, then
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-18.html:2104]: as written in the cp change this line in:
If the Document Consumer does support the On-Demand Documents Option,

File: ITI/TF/Volume2/ITI-18.html [hunk 5]
```diff
@@ -2033,26 +2088,56 @@ <h6 id="3.18.4.1.2.4">3.18.4.1.2.4 Stored Query IDs</h6>
...
+                                to specify the $targetCommunityIdList parameter containing the homeCommunityIds of desired responding communities. 
+                                When the Initiating Gateway receives the Registry Stored Query transaction from the Document Consumer in which 
+                                the “$targetCommunityIdList” parameter is specified:
+                                <ul>
+                                    <li class="bullet-list1">If it supports the “Target Communities” Option, it shall accept and process the query
+                                        accordingly, addressing it to the communities identified by the homeCommunityIds contained in the parameter.
+                                    </li>
+                                    <li class="bullet-list1">○	If it doesn’t support the “Target Communities” Option, will ignore the parameter 
+                                        and manage the query as if it is not used. The Document Consumer shall be aware of the possibility that it 
+                                        will get more results than is expected (See Section 3.38.4.1.3.1).
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-18.html:2135]: suggest to put an html reference on Section 3.38.4.1.3.1

**Review by gregoriocanal — CHANGES_REQUESTED** [review]
see comments

**Comment by JohnMoehrke** [comment]
@gregoriocanal your recommendations seem reasonable. Can you make the updates to this PR?

**Review by gregoriocanal — APPROVED** [review]
updated as requested

**Pull request merged**